lib/Importer/TFLiteModelLoader.cpp

```diff
@@ -1968,6 +1971,46 @@ Error TFLiteModelLoader::loadSlice(const tflite::Operator *op,
   return setOutputNodeValue(op, output);
 }
 
+Error TFLiteModelLoader::loadResizeBilinear(const tflite::Operator *op,
+                                            const OperatorInfo &opInfo) {
+  NodeValue input;
+  ASSIGN_VALUE_OR_RETURN_ERR(input, getInputNodeValue(op, 0));
+  NodeValue size;
+  ASSIGN_VALUE_OR_RETURN_ERR(size, getInputNodeValue(op, 1));
+
+  Constant *sizeC = llvm::dyn_cast<Constant>(size.getNode());
+  RETURN_ERR_IF_NOT(sizeC, opErrMsg(opInfo, "Size input should be a constant"));
+
+  const auto *opts = op->builtin_options_as_ResizeBilinearOptions();
+
+  bool align_corners = opts->align_corners();
+  if (align_corners) {
+    LOG(WARNING) << "Resize Bilinear align corners option is ignored.";
```

Comment:
Please use the ```opErrMsg``` utility to format the error/warning string to provide more context.